### ISSUE TITLE
IsRequired

### ISSUE THREAD

**Issue by FBiDev** [ChrisHodges/ExcelToEnumerable#21]
when use IsRequired after using UsesColumnNamed
he define IsRequired to all  Cells that use UsesColumnNamed

i put a example here:

var op = new ExcelToEnumerableOptionsBuilder<Favorecido>();
op.Build();
op.IgnoreColumsWithoutMatchingProperties();
op.AllPropertiesOptionalByDefault();

op.Property(p => p.CodigoDocumento).UsesColumnNamed("CodigoDocumentoX");
op.Property(p => p.EndBairro).UsesColumnNamed("Bairro");
op.Property(p => p.EndBairro).IsRequired();
arquivo.Path.ExcelToEnumerable<Favorecido>(op);

he accused that "CodigoDocumentoX" is Required, but i put in EndBairro

**Comment by miguel-fernandez-corral** [comment]
Yes, I have the same issue. Every property is required too. It's seem that the packaged **nuget** version is not aligned with the code itself. I cant see the namespace **ExcelToEnumerable.Attributes**.
